prosper/crons/FetchMapInfo.py

```diff
@@ -0,0 +1,100 @@
+'''FetchMapInfo.py: a script for saving the map statistics from EVE Online XML API'''
+
+from os import path #TODO: path->plumbum?
+
+import requests
+import pandas
+import xml.etree.ElementTree as ET
```

Comment:
Will prefer to use [eveapi](https://github.com/ntt/eveapi) library for fetching data